.github/workflows/pages.yml

```
@@ -0,0 +1,39 @@
+name: Deployment
+# This job builds and deploys atlas to github pages.
+# It runs on every push to master.
+on:
+  push:
+    branches:
+      - master
```

```suggestion
      - main
```

We don't use that verbiage here ;-)